docs/whatsnew_1.5.7.html

```diff
@@ -9,10 +9,9 @@
      */
 //-->
 <head>
-    <title>What's New in Zen Cart&reg; v1.5.7</title>
     <meta charset="utf-8">
+    <title>What's New in Zen Cart&reg; v1.5.7</title>
```

Comment:
Suggest change to &tm; here.  I'm trying to do this in other places.

Reply:
Don't you mean
`&trade;`
?

Reply:
yes thanks.